newIDE/app/src/MainFrame/EditorContainers/HomePage/BuildSection/index.js

```diff
@@ -252,13 +281,28 @@ const BuildSection = ({
     >
       <SectionRow>
         <GridList
-          cols={getItemsColumns(windowWidth)}
+          cols={columnsCount}
           style={styles.grid}
           cellHeight="auto"
           spacing={cellSpacing}
         >
           {examplesAndTemplatesToDisplay.gridItems}
         </GridList>
+        <Line justifyContent={'center'}>
+          <FlatButton
+            primary
+            disabled={examplesAndTemplatesToDisplay.hasReachedEnd}
+            fullWidth={isMobile}
+            label={
+              examplesAndTemplatesToDisplay.hasReachedEnd ? (
+                <Trans>Browse all examples in the Learn tab</Trans>
+              ) : (
+                <Trans>See more</Trans>
+              )
+            }
+            onClick={() => setPageIndex(pageIndex_ => pageIndex_ + 1)}
```

Comment:
is that postVariableUnderscore on purpose? first time I'm seeing this

Reply:
I always wonder if it's at the start or at the end that it should be put 😅